code/code/misc/spells.h

```diff
@@ -591,6 +591,7 @@ enum spellNumT {
      SKILL_BULLYWUGCROAK,
      SKILL_AVIAN,
      SKILL_FISHBURBLE,
+     SKILL_COMMON,
```

Comment:
Will injecting an item into the middle of an enum break learnedness data in player files?